app/code/Magento/Cms/Model/Wysiwyg/Images/Storage.php

```diff
@@ -197,6 +204,7 @@ public function __construct(
         array $data = [],
         \Magento\Framework\Filesystem\DriverInterface $file = null,
         \Magento\Framework\Filesystem\Io\File $ioFile = null,
+        \Magento\Framework\File\Mime $mime = null,
```

Comment:
The new optional parameter should be added to the end of the parameters list. Please move it after `\Psr\Log\LoggerInterface $logger = null`

Reply:
I've moved the optional parameter at the end of the parameter list